conf/service/application.conf

```diff
@@ -25,7 +25,7 @@ topologySpecFactory.topologyNames=localGobblinCluster
 topologySpecFactory.localGobblinCluster.description="StandaloneClusterTopology"
 topologySpecFactory.localGobblinCluster.version="1"
 topologySpecFactory.localGobblinCluster.uri="gobblinCluster"
-topologySpecFactory.localGobblinCluster.specExecutorInstanceProducer.class="gobblin.service.SimpleKafkaSpecExecutorInstanceProducer"
+topologySpecFactory.localGobblinCluster.specExecutorInstance.class="gobblin.service.SimpleKafkaSpecProducer"
```

Comment:
+1 I would recommend to update it as well, the code review feedback will be lost / suppressed if we review the PR in current shape because the files physical path will also change. Sorry for the extra work, I owe you one :)